### PR TITLE
Sphinx docs

### PR DESCRIPTION
**Pull request by mixxorz**
Here's the initial docs for `behave-django` on Sphinx. Is it missing anything? #16 

Update: [Here's what it looks like.](https://pythonhosted.org/behave-django/)

### CODE REVIEW

**Comment by bittner** [comment]
Nice! You're using the same styles as the [behave docs](http://pythonhosted.org/behave/)! Beautiful!

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+# -- Options for HTML output ----------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+html_theme = 'alabaster'
+
+# Theme options are theme-specific and customize the look and feel of a theme
+# further.  For a list of options available for each theme, see the
+# documentation.
+#html_theme_options = {}
```
Comment (nikolas) [inline docs/conf.py]: If you want, you can add the "fork me on github" banner that links back to this repo here like this:

``` py
html_theme_options = {
    'github_user': 'mixxorz',
    'github_repo': 'behave-django',
    'github_banner': 'true',
}
```

**Comment by nikolas** [comment]
I like the alabaster theme too. I'm using it for this project I'm working on: http://django-pagetree.readthedocs.org/

**Comment by mixxorz** [comment]
Honestly, that's just what `sphinx-quickstart` gave me. :stuck_out_tongue: 
I'll add the GitHub banner and I'll merge this in. I'll also update the README and other things.

**Pull request merged**